modules/engine/src/testing/listeners.spec.ts

```diff
@@ -162,15 +169,41 @@ describe(testName, () => {
     it("should work if responder is alice (submit withdraw to chain + resolve transfer)", () => {});
     it("should work if responder is bob (DONT submit withdraw to chain + resolve transfer)", () => {});
   });
-  describe.skip("withdrawal resolution", () => {
-    it("should not handle transfers", () => {});
-    it("should fail if getting transfer from store fails", () => {});
-    it("should fail if there is no transfer in store", () => {});
-    it("should work for initiator", () => {});
-    it("should fail if responder cannot addSignatures to commitment", () => {});
-    it("should fail if responder cannot save commitment", () => {});
-    it("should should work if responder is alice (submit withdraw to chain)", () => {});
-    it("should should work if responder is bob (DONT submit withdraw to chain)", () => {});
+
+  describe("withdrawal resolution", () => {
+    it("should not create commitments for cancelled withdrawals", async () => {
+      const alice = getRandomChannelSigner();
+      const bob = getRandomChannelSigner();
+      const test = createTestChannelState("create", { alice: alice.address, bob: bob.address });
+      test.transfer.transferState.fee = "0";
+      test.transfer.transferState.nonce = "0";
+      test.transfer.transferState.initiatorSignature = mkSig("0x0");
```

Comment:
This will never be the case, the initiator will always have to supply a valid signature